packages/lab/src/Flow/stories/Usage.mdx

```diff
@@ -132,17 +132,17 @@ The parameters and the list of inputs and outputs are passed as a property of th
 Find an example of a custom node below.
```

Comment:
I think we also need to review the "Sidebar" and "Custom nodes and node types" sections since they are not up to date with these changes.